batch/batch/front_end/validate.py

```diff
@@ -39,17 +39,18 @@
 # image -> process/image
 # mount_docker_socket -> process/mount_docker_socket
 # pvc_size -> resources/storage
+# gcsfuse -> cloudfuse
 
 
 job_validator = keyed(
     {
         'always_run': bool_type,
         'attributes': dictof(str_type),
         'env': listof(keyed({'name': str_type, 'value': str_type})),
-        'gcsfuse': listof(
+        'cloudfuse': listof(
             keyed(
                 {
-                    required('bucket'): non_empty_str_type,
+                    required('location'): non_empty_str_type,
```

Comment:
let's just keep this as bucket. Location is pretty overloaded in our code-base already. I know "bucket" isn't really an Azure term, but I think it's clear that it means a fully qualified container (i.e. "$account/$container").